### DISCUSSION TITLE
Disable autocomplete for specific inputs

### DISCUSSION THREAD

**Discussion by paulvales** [fomantic/Fomantic-UI#940] [Elements (button, image, label, loader, ...)] [answered]
Hi! maybe disable autocomplete for calendar or add option for all inputs http://joxi.ru/zANeR3gtv9b8L2

**Comment by lubber-de** [comment, marked as answer]
This is a browser related feature.
As `input` is not a module you can try adding
`autocomplete="off" ` to either its Form or element directly. Same for `calendar` (or `dropdown` or `search`) , just modify the embedded input field accordingly 

But beware, it does not work on all input fields. 
See https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion
